### PR TITLE
Add a link to Help site

### PR DESCRIPTION
**Pull request by Daniel-Weng-TC1**
References AB#415977

Add help icon to ModusNavbar in stencil-workspace. My 1st PR may miss some steps.

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (including CHANGELOG updates)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ View all releases at: https://github.com/trimble-oss/modus-web-components/releas
 
 ## Unreleased
 
+## 0.1.19 - 2022-19-09
```
Comment (ryan-henness-trimble) [inline CHANGELOG.md:9]: Looking good. There are some Storybook docs that should be updated in `stencil-workspace\storybook\stories\components\modus-navbar` to include the additions to the new props

These docs haven't been updated in a while, the `.mdx` file still contains the hand-written tables for the properties/dom events. These tables can be safely replaced with a copy from the generated docs at `stencil-workspace\src\components\modus-navbar\readme.md`

Site note: you will have to `npm install` inside of the `storybook` directory, and then run `npm storybook` to get the site started

**Comment by Daniel-Weng-TC1** [comment]
Just updated storybook a bit, so let me know any further changes

**Comment by coliff** [comment]
Heya @Daniel-Weng-TC1 - FYI: there was an issue with the Azure Static Web Apps Deploy Preview not working (not your fault!).. which I'm working on fixing right now. sorry for the trouble!
I think it was caused by having 10x deploy preview branch URLs active which is the limit which meant new PRs were failing. I just deleted all the old ones we no longer need and re-kicked off the failed Azure CI/CD now.

**Comment by Daniel-Weng-TC1** [comment]
Thanks!

**Review by ryan-henness-trimble — DISMISSED** [review]

**Comment by Daniel-Weng-TC1** [comment]
Squashed and rebased

**Review by ryan-henness-trimble — APPROVED** [review]

**Comment by Daniel-Weng-TC1** [comment]
@ryan-henness-trimble Can you help me resolve "Unresolved conversations"?

**Pull request merged**